test/integration/api/assignments.js

```diff
@@ -473,20 +478,23 @@ describe('/projects/:id/assignments', () => {
             .expect(403))))));
 
     it('should return notfound if the project does not exist', testService((service) =>
-      service.login('bob', (asBob) =>
-        asBob.get('/v1/users/current').expect(200).then(({ body }) => body.id)
-          .then((bobId) => service.delete('/v1/projects/99/assignments/manager/' + bobId)
-            .expect(404)))));
+      service.login('alice', (asAlice) =>
+        service.login('bob', (asBob) =>
```

Comment:
Doesn't need to change, but just FYI, you can pass an array to `service.login()`:

```js
service.login(['alice', 'bob'], (asAlice, asBob) =>
```